src/pages/requests/[requestId]/conversation/[conversationId]/index.svelte

```diff
@@ -0,0 +1,6 @@
+<script>
+import Request from '../../../../../components/Request.svelte'
```

Comment:
indentation

Reply:
There's an unfortunate mix of indentation of <script> code throughout. Probably worth a separate PR just for whitespace cleanup.